### ISSUE TITLE
inaccessible home folder?

### ISSUE THREAD

**Issue by respeccing** [thestinger/playpen#59]
**tl;dr:** when using playpen arg `--user=` then that user's homedir is inaccessible and seen as `root:root` and `drwxrwxrwt`
Perhaps this is intended or I'm missing something obvious.
**EDIT:** looks like it's intended:
>A mount namespace is leveraged to provide writable /tmp, /dev/tmp and home directories as in-memory (tmpfs) filesystems. Since these mounts are private, any number of Playpen instances can share the same root.

`none on /home/manj1 type tmpfs (rw,nosuid,nodev,relatime)`

Observe permissions for home folder named `manjaro`

What the actual permissions are:
```bash
[manjaro@manjaro playpen]$ sudo ./playpen ./sandbox --user=nobody --syscalls-file=./whitelist -- ls -la /home/
total 0
drwxr-xr-x 3 root    root     60 Mar 28 07:29 .
drwxrwxrwt 2 root    root     40 Mar 28 07:54 ..
drwxr-xr-x 2 manjaro manjaro 100 Mar 28 07:48 manjaro
```

The same folder when using the username `manjaro` is inaccessible now:

```bash
[manjaro@manjaro playpen]$ sudo ./playpen ./sandbox --user=manjaro --syscalls-file=./whitelist -- ls -la /home/
total 0
drwxr-xr-x  3 root    root     60 Mar 28 07:29 .
drwxr-xr-x 16 manjaro manjaro 400 Mar 28 07:24 ..
drwxrwxrwt  2 root    root     40 Mar 28 07:54 manjaro
[manjaro@manjaro playpen]$ date
Mon Mar 28 07:54:47 UTC 2016
```

Why that is bad is because no files can be seen when running as the intended user `manjaro`:

```bash
[manjaro@manjaro playpen]$ sudo ./playpen ./sandbox --user=manjaro --syscalls-file=./whitelist -- ls -la /home/manjaro/
total 0
drwxrwxrwt 2 root root 40 Mar 28 07:56 .
drwxr-xr-x 3 root root 60 Mar 28 07:29 ..
[manjaro@manjaro playpen]$ sudo ./playpen ./sandbox --user=nobody --syscalls-file=./whitelist -- ls -la /home/manjaro/
total 12
drwxr-xr-x 2 manjaro manjaro 100 Mar 28 07:48 .
drwxr-xr-x 3 root    root     60 Mar 28 07:29 ..
-rw------- 1 manjaro manjaro  81 Mar 28 07:38 .bash_history
-rw-r--r-- 1 manjaro manjaro  19 Mar 28 07:45 .bashrc
-rwxr-xr-x 1 manjaro manjaro  40 Mar 28 07:48 boo.sh
```

The playpen version tested was latest commit: f8becccff56827c1d9ed70f0edc9470bf408bb86

User was created by:
```
sudo chroot sandbox/
useradd manjaro
mkdir home/manjaro
chown manjaro.manjaro /home/manjaro/
```
(I know I should've specified `--create-home` to `useradd` instead! but I forgot; I've just tested this with a new user, but the same chown to root:root and drwxrwxrwt attrs for its home dir happen when --user=manj1 is passed to playpen, thus its homedir is again inaccessible.)
